week-03/README.md

```diff
@@ -438,6 +450,8 @@ b) if that abbreviation is recognized, a tooltip displays the expanded text; Pre
 - Question (lceballos-stat6250): What is the difference between the eq function and "="?
 - Question (ldeng11−stat6250): What is the running sequence when you have different logical operand in the same statement?
 - *Question* (akrishnamurthy-stat6250): How are compound expressions evaluated in SAS ?
+- *Answer* (asharda-stat6250): AND & and, both. If both expressions are true, then the compound expression is true.
+OR | or, either. If either expression is true, then the compound expression is true. 
```

Comment:
This line break needs to be removed before this PR can be merged. Please update and DM me in Slack with PR URL for re-evaluation.